alf/layers.py

```diff
@@ -3612,3 +3612,34 @@ def forward(self, input):
                 lambda x: x.float() if x.dtype.is_floating_point else x, input)
         with torch.cuda.amp.autocast(self._enabled):
             return self._net(input)
+
+
+class MultiHeadAttention(nn.Module):
```

Comment:
This is not the standard MultiHeadAttention in transformer. Perhaps using a different name (e.g. SimpleAttention)?

Reply:
Changed name to SimpleAttention to avoid confusion.